.github/workflows/snapshoty.yml

```
@@ -0,0 +1,47 @@
+---
+# Publish a snapshot. A "snapshot" is a packaging of the latest *unreleased* APM agent,
+# published to a known GCS bucket for use in edge demo/test environments.
+name: Snapshoty
+
+on:
+  push:
+    branches:
+    - main
+    paths-ignore:
+    - '*.md'
+    - '*.asciidoc'
+    - 'docs/**'
+
+env:
+  MAVEN_CONFIG: "-V -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn -Dhttps.protocols=TLSv1.2 -Dmaven.wagon.http.retryHandler.count=3 -Dmaven.wagon.httpconnectionManager.ttlSeconds=25"
```

Just a small suggestion for readbility and maintainability.

```suggestion
  MAVEN_CONFIG: >-
    -V
    -B
    -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
    -Dhttps.protocols=TLSv1.2 -Dmaven.wagon.http.retryHandler.count=3
    -Dmaven.wagon.httpconnectionManager.ttlSeconds=25
```